packages/calcite-components/src/components/tabs/tabs.e2e.ts

```diff
@@ -374,15 +374,15 @@ describe("calcite-tabs", () => {
       await page.setContent(html`
         <calcite-tabs>
           <calcite-tab-nav slot="title-group">
-            <calcite-tab-title id="tab-title-1" closable>Tab 1 Title</calcite-tab-title>
-            <calcite-tab-title id="tab-title-2" closable>Tab 2 Title</calcite-tab-title>
-            <calcite-tab-title id="tab-title-3" closable>Tab 3 Title</calcite-tab-title>
-            <calcite-tab-title id="tab-title-4" closable selected>Tab 4 Title</calcite-tab-title>
+            <calcite-tab-title id="tab-title-1" data-name="1" closable>Tab 1 Title</calcite-tab-title>
```

Comment:
`tab`s and `tab-title`s already have `id` set. Could you use that instead of adding an extra attribute?